core/misc/announce.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Adds an HTML element and method to trigger audio UAs to read system messages.
+ *
+ * Use Backdrop.announce() to indicate to screen reader users that an element on
```

Comment:
It might be helpful to also mention here, that the library needs to get included with something like `backdrop_add_library('system', 'backdrop.announce')` or `$form['#attached']['library'][] = array('system', 'backdrop.announce');`

Not sure, how detailed this should be.